src/PlanView/SurveyItemEditor.qml

```diff
@@ -398,6 +402,10 @@ Rectangle {
                         id:                 presetNameField
                         Layout.fillWidth:   true
                     }
+                    QGCLabel {
+                        id:                 presetNameError
+                        color:              "red"
```

Comment:
The color here should use QGCPalette `warningText` color instead of a hardcoded color.